dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/BoulderContainer.java

```diff
@@ -0,0 +1,253 @@
+package com.ibm.ws.security.acme.docker;
+
+import java.io.IOException;
+import java.security.cert.X509Certificate;
+import java.time.Duration;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.http.StatusLine;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.MountableFile;
+
+import com.github.dockerjava.api.model.ContainerNetwork;
+import com.github.dockerjava.api.model.ContainerNetwork.Ipam;
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.acme.utils.AcmeFatUtils;
+
+public class BoulderContainer extends CAContainer {
+
+	public static final int HTTP_PORT = 5002;
+
+	/** The port used to listen for incoming ACME requests. */
+	public static final int LISTEN_PORT = 4001;
+
+	/** The REST management API port. */
+	public static final int MANAGEMENT_PORT = 8055;
```

Comment:
It appears that you set this value in CAContainer... do we need this one? This value is somewhat ghosting the one in CAContainer.